src/man/sssd-ad.5.xml

```diff
@@ -109,6 +109,10 @@ ldap_id_mapping = False
             case-insensitive in the AD provider for compatibility with Active
             Directory's LDAP implementation.
         </para>
+        <para>
+            SSSD only resolves Security-enabled Active Directory group types.
+            (i.e. Not <quote>distribution</quote> groups)
+        </para>
```

Comment:
Hi,

I think `Security-enabled` is not common Active Directory speak in this context. I would suggest something like `SSSD only resolves Active Directory Security Groups` (since `Security Group` is used in AD User and Computers in the user and group listing) or `SSSD only resolves Active Directory groups of group type "Security"`(since this is shown in the properties of a group). @abbra, do you agree or do you have other suggestions?

We already have a reference to MSFT documentation in the `sssd-ad` man page, so I think it might worth to add https://docs.microsoft.com/en-us/windows-server/identity/ad-ds/manage/understand-security-groups here as well.

bye,
Sumit

Reply:
I'd prefer `Security Groups` but you need to add more details around 'Domain-Local' scope, I think. It becomes a bit complex to explain ;)

Reply:
Hi,

thanks, adding some info about how domain-local groups are handled is a good idea. @justin-stephenson, can you add it or do you need more context?

bye,
Sumit

Reply:
There is mention of how SSSD handles domain local groups in the option description for `ad_allow_remote_domain_local_groups`. Is it enough or should it be mentioned in this part as well?

Reply:
Hi,

I would copy the content into this part as well.

bye,
Sumit

Reply:
Updated, please check. Thank you.